gui/wxpython/startup/guiutils.py

```diff
@@ -210,6 +210,22 @@ def create_mapset_interactively(guiparent, grassdb, location):
     dlg.Destroy()
     return mapset
 
+def create_location_interactively(guiparent, grassdb):
+    """
+    Location wizard started
+    """
+    from location_wizard.wizard import LocationWizard
+    gWizard = None
+    try:
+        gWizard = LocationWizard(parent=guiparent,
+                                 grassdatabase=grassdb)
+    except OSError as err:
+            GError(
+                parent=guiparent,
+                message=_("Unable to create new location %s") % err,
+                showTraceback=False,
+            )
+    return gWizard
```

Comment:
I can see this would be nice, but you need to move lot more functionality here to make it useful. This will also help you clean up couple calls to non-existent or wrong methods in `OnCreateLocation()` in `tree.py`.

You may need to return tuple (1) or something like that from this function to get all what you need out, but returning the wizard seems inconsistent with the other functions you created where the beauty is that we don't need to deal with any GUI afterwards and we get just the data.

(1) A tuple is not ideal, but easiest to write, so great for prototype. Alternatively, a dict is of course the Python's programmer unfortunate default for everything ([Python: What if everything was a dict?](https://argumate.tumblr.com/post/118013166244/python-what-if-everything-was-a-dict-java-what)).

Reply:
Now it returns the tuple: (gWizard.grassdatabase, gWizard.location, and either PERNAMENT or user mapset).